### ISSUE TITLE
Travis log too long

### ISSUE THREAD

**Issue by martinpopel** [tensorflow/tensor2tensor#523]
Travis CI has 4MB limit on the log file, but current T2T tests exceed this limit.
As a result I cannot see the real cause why a [given build](https://travis-ci.org/tensorflow/tensor2tensor/jobs/329572628) failed.
Most of the log (if downloaded as a raw file) is full of download progress bars, e.g.
```
  Downloading mpmath-1.0.0.tar.gz (511kB)
[?25l
[K    2% |▋                               | 10kB 53.6MB/s eta 0:00:01
[K    4% |█▎                              | 20kB 41.3MB/s eta 0:00:01
[K    6% |██                              | 30kB 42.5MB/s eta 0:00:01
[K    8% |██▋                             | 40kB 41.3MB/s eta 0:00:01
[K    10% |███▏                            | 51kB 42.6MB/s eta 0:00:01
```
This is the first time I see this problem, but it seems that until a new pip version with [--progress-bar off](https://github.com/pypa/pip/pull/4194) option is released, we should use [a workaround](https://github.com/pypa/pip/issues/2756#issuecomment-268773883) `pip install package | cat && exit ${PIPESTATUS[0]}` or `set -o pipefail; pip install package | cat`.

**Comment by rsepassi** [comment]
how about `pip install -q` to have the install be quiet?

**Comment by rsepassi** [comment]
And `git clone -q`

**Comment by rsepassi** [comment]
And thank you for spotting this.

**Comment by martinpopel** [comment]
According to https://github.com/pypa/pip/issues/2756 if we provide the `--quiet` option, pip does not display exceptions, which is a bit unfortunate for Travis. That said, it would be still better than the current state.
And yes, `git clone -q` is a good idea.

Unfortunately, it seems Travis is now completely broken. See e.g. https://travis-ci.org/tensorflow/tensor2tensor/builds/329575693 (note that a given commit just changes README.md).
In https://github.com/tensorflow/tensor2tensor/pull/524 I have tried to silence the pip, but Travis again failed for other reasons.

**Comment by rsepassi** [comment]
I think it’s fine not to see the pip exceptions in Travis. The focus of the
tests is not to test pip install. If a pip command fails, we'll see it and
can retry locally to see the error.

**Comment by martinpopel** [comment]
I changed `| cat && exit ${PIPESTATUS[0]}` into `pip install -q` in https://github.com/tensorflow/tensor2tensor/pull/524/commits/61005b0b1e063e3e26cb403cf6e8cdea3509c3b5
This commit (most probably disregarding its content) caused [Travis rebuild](https://travis-ci.org/tensorflow/tensor2tensor/builds/330535879) and now it work and the raw log has just 82 KiB.
I was not able to find a place where to set `git clone -q` (`git clone` is done automatically by Travis).
So this issue can be closed once #524 is merged (sorry for combining two topics in one PR).
